atomic-design.md

```diff
@@ -28,7 +28,7 @@ An atomic element is any self contained code pattern that follows the Atomic Des
 
 ### Templates
 
-- To us: literally what atoms, molecules and organisms live on the wagtail templates
+- To us: literally which atoms, molecules and organisms live on the wagtail templates
```

Comment:
This line is still a bit unclear, overall. How about something like this: "Templates are types of pages that have a particular layout. The primary ones are Landing, Sublanding, Browse, Learn, and Document Detail. They are differentiated primarily their major layout features, such as whether or not they have a hero, whether the sidebar is on the right or left, etc. Each template allows certain atoms, molecules, and organisms to be included in its major sections."